docs/tutorial/commands/index.md

```diff
@@ -257,6 +257,39 @@ Commands:
 
 </div>
 
+
+## Sorting of the commands
+
+Note that by design, **Typer** shows the commands in the order they've been declared.
```

Comment:
Note: if we keep the current behavour of command/group sorting, then we probably want to comment on that in the docs as well, because otherwise it may be confusing. I can add this after we decide how to move forward.